components/summary/summary--group.hbs

```diff
@@ -55,6 +55,22 @@
       </div>
     </div>
 
+    <div class="summary__item">
+      <div class="grid">
+        <div class="grid__col col-12@xs col-10@m">
+          <div class="summary__question mars">Description</div>
+        </div>
+        <div class="grid__col col-4@xs col-2@m push-8@xs nopull-@m">
```

Comment:
The change link plays back in Voiceover before the answer has been read back.

All other answers play back in the following order:
> Question > Answer > Change answer link

Maintaining order is important for comprehension and cognitive load for users relying on Voiceover.